### PR TITLE
Added Navigation Menu, ValueinfoFields and In/OutputFields

### PR DESCRIPTION
**Pull request by SergeyMOV**

### CODE REVIEW

File: components/In&OutputFields.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+export default function Valuefields() {
+  return (
+    <Buttonsection>
+      <button>+50€</button>
+      <br></br>
```
Comment (stefankorder) [inline components/In&OutputFields.js]: Don't use `<br>` to create space between your buttons. Use styling with styled components instead.

File: components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @next/next/no-html-link-for-pages */
```
Comment (stefankorder) [inline components/Navbar.js]: You really need to deactivate eslint at this point?

**Review by stefankorder — CHANGES_REQUESTED** [review]
Looks good so far! Take a look at my comments and check again if your html structure is that good.

File: components/Navbar.js (hunk 2 above)
Comment (SergeyMOV) [reply components/Navbar.js]: No, I just tried something there and forgot to delete it.

**Review by actyralf — APPROVED** [review]
👍

**Pull request merged**